src/js/common/schemaform/SaveInProgressErrorPage.jsx

```diff
@@ -78,10 +78,21 @@ class SaveInProgressErrorPage extends React.Component {
       case LOAD_STATUSES.failure:
         content = (
           <div>
-            <div className="usa-alert usa-alert-error no-background-image">We’re sorry, but something went wrong. Please try applying again in a few moments.</div>
+            <div className="usa-alert usa-alert-error no-background-image">We’re sorry. We’re having some server issues and are working to fix them. Please try applying again in a few moments.</div>
             <div style={{ marginTop: '30px' }}>
               {this.getBackButton()}
-              <button className="usa-button-primary" onClick={this.reloadForm}>Resume previous application</button>
+              <button className="usa-button-primary" onClick={this.reloadForm}>Continue Application</button>
+            </div>
+          </div>
+        );
+        break;
+      case LOAD_STATUSES.invalidData:
+        content = (
+          <div>
+            <div className="usa-alert usa-alert-error no-background-image">We’re sorry. Something went wrong when we tried to access your application. We’re working to fix this. You can try applying again in a few moments or start your application over.</div>
+            <div style={{ marginTop: '30px' }}>
+              {this.getBackButton()}
+              <button className="usa-button-primary" onClick={this.reloadForm}>Continue Application</button>
```

Comment:
Would we want to provide a button option to start over since starting over is mentioned in the error message?

Reply:
I'm not sure we want to encourage starting over. @stacyhlinn, should we show the start over button on this error page? Or maybe change the text to mention that they need to click Back first to start over?